solutions/standard/variables.tf

```diff
@@ -0,0 +1,310 @@
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of a new or the existing resource group to provision the client to site VPN. If a prefix input variable is passed, it is prefixed to the value in the `<prefix>-value` format."
+}
+
+variable "use_existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "name" {
+  type        = string
+  description = "The name of the VPN."
+  default     = "cts-vpn"
+}
+
+##############################################################################
+# Secrets Manager resources
+##############################################################################
+
+variable "existing_secrets_manager_instance_crn" {
+  type        = string
+  description = "The CRN of existing secrets manager where the certificate to use for the VPN is stored or where the new certificate will be created."
+}
+
+variable "existing_secrets_manager_cert_crn" {
+  type        = string
+  description = "The CRN of existing secrets manager private certificate to use to create VPN. If the value is null, then new private certificate is created."
+  default     = null
+}
+
+variable "existing_secrets_manager_secret_group_id" {
+  type        = string
+  description = "The CRN of existing secrets manager secret group id used for new created certificate. If the value is null, then new secrets manager secret group is created."
+  default     = null
+}
+
+variable "cert_common_name" {
+  type        = string
+  description = "A fully qualified domain name or host domain name for the certificate to be created.  Only used when `existing_secrets_manager_cert_crn` is `null`."
+  default     = null
+}
+
+variable "certificate_template_name" {
+  type        = string
+  description = "The name of the Certificate Template to create for a private_cert secret engine. When `existing_secrets_manager_cert_crn` is `null`, then it has to be the existing template name that exists in the private cert engine."
+  default     = null
+}
+
+##############################################################################
+# client-to-site VPN
+##############################################################################
+
+variable "vpn_subnet_cidr_zone_1" {
+  type        = string
+  description = "The CIDR range to use from the first zone in the region (or zone specified in the 'vpn_zone_1' input variable)"
+  default     = "10.10.40.0/24"
+}
+
+variable "vpn_subnet_cidr_zone_2" {
+  type        = string
+  description = "The CIDR range to use from the second zone in the region (or zone specified in the 'vpn_zone_2' input variable)."
+  default     = "10.10.80.0/24"
+}
+
+variable "network_acls" {
+  description = "The list of ACLs to create. Provide at least one rule for each ACL."
```

Comment:
This input is too complex for consumers to figure out in the Projects UI. Your going to have to add supporting markdown documentation to show how it can be used, and link to it in the variable descrption. Similar to this: https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/main/solutions/standard/DA-keys.md

Reply:
should this have a default value?

Reply:
As per call, lets convert to 1 required string variable for inbound source address. Everything else we can programmtcally determine

Reply:
added new input variable `remote_cidr`